doc/api-asyncio/cursors.rst

```diff
@@ -5,7 +5,7 @@
 
 .. autoclass:: AsyncIOMotorCursor
   :members:
-
+  :inherited-members:
```

Comment:
Does this include anything that `members` doesn't? I don't see a difference in the rendered docs. FYI to get the docs to build with `inherited-members` add this workaround:
```
diff --git a/doc/motor_extensions.py b/doc/motor_extensions.py
index 16cbe11..75d1a13 100644
--- a/doc/motor_extensions.py
+++ b/doc/motor_extensions.py
@@ -129,7 +129,12 @@ def get_motor_attr(motor_class, name, *defargs):
     attribute. While we're at it, store some info about each attribute
     in the global motor_info dict.
     """
-    attr = safe_getattr(motor_class, name)
+    try:
+        attr = safe_getattr(motor_class, name)
+    except AttributeError:
+        if name == '__annotations__':
+            return {}
+        raise
```

Reply:
Without `:inherited-members:`, I am not seeing docs for `fetch_next` and `next_object` in my local build.

Also, without `:inherited-members:` the doctests are not run:
```
Doctest summary
===============
  108 tests
    0 failures in tests
    0 failures in setup code
    0 failures in cleanup code
build succeeded.
```

With `:inherited-members:`:
```
Doctest summary
===============
  164 tests
    0 failures in tests
    0 failures in setup code
    0 failures in cleanup code
build succeeded.
```

Some tests are getting run multiple times (because e.g. of multiple classes inheriting from the same base class with a doctest) which is why there is such a large disparity in the number of test cases

Reply:
Thanks for the workaround! I couldn't figure that out. I also have no clue why it even happens - it seems to be `tox`-specific because running `make html` or `make doctest` in `doc/` doesn't cause this failure.

Reply:
FYI this is where the `__annotations__` error comes from: https://github.com/sphinx-doc/sphinx/blob/v3.2.0/sphinx/ext/autodoc/__init__.py#L624